api/locations.md

```diff
@@ -86,58 +92,71 @@ Parameter | Description
 
 ### Return values
 
-Available parking space types are:
-
-* `off-street` Off-Street parking space. _Default_
-* `park-and-ride` Park & Ride (P+R)
-* `professional-garage` Professional garage
-* `airport-parking` Airport-Parking
-* `harbor-parking` Harbor-Parking
-
-Available options for a parking space are:
-
-* `excess length` The parking space can be used by cars with a small trailer.
-* `indoor` The parking space has a roof or can be an underground garage.
-* `disabled` Parking space is accessible for disabled people.
-* `patio` The parking space is located at an inner courtyard.
-* `barrier` Parking space has a barrier and user needs a smartphone to open it.
-* `charging station` Parking space with an electric vehicle charging station.
-* `key_necessary` Parking space can only be accessed with a key.
-* `long_term_booking` Parking space is bookable for longer time period.
-* `surveillance` Parking space has surveillance.
-* `valet_service` Parking space contains valet service.
-
-`user_owned` is `true` if the user who requested the location is also the owner of the parking space. This allows the web and the apps to determine if the user could block the parking space from bookings. The blocking feature itself is not yet supported in APIv4.
-
-`price` and `price_period`: The `price` is the actual price shown to the user. The `price_period` contains the period for how long the price is valid:
-
-* `day`
-* `week`
-* `month`
-* `timeframe`
-
-`pricing` contains information about the pricing and price prediction. It contains the keys:
-
-* `rate_hourly`
-* `rate_daily`
-* `rate_weekly`
-* `rate_monthly`
+#### HTTP Codes
+Code | Description
+---- | -----
+200  | location found, body contains description
+404  | location was not found, check the location id and that your user has the correct access rights to see the location
+422  | request validation failed because the request data was malformed
+
+#### Body
+Field             | Element                | Type         | (Element and) Description
+----------------- | ---------------------- | ------------ | -----
+`address_slug`    |                        | `string`     | street name and number of the location in url address format
+`availability`    |                        | `dictionary` | describes the availability of the location in relation to the request
+                  | `current`              | `number`     | the currently available number of parking spaces at the location for the requested time period defined by `start` and `end`
```

Comment:
I would remove "currently available" and change it to something like "available for timeframe"